.github/workflows/default.yml

```diff
@@ -77,14 +77,40 @@ jobs:
           gradle-version: ${{ matrix.gradle }}
           arguments: formatKotlin lintKotlin
 
+      - name: Assert fixes
+        run: |
+          cd test-project/build/reports/ktlint
+          grep 'no-empty-class-body' main-format.txt | grep -q 'EmptyClassBodyClass.kt:3:27'
+          grep 'op-spacing' test-format.txt | grep -q 'OpSpacing.kt:5:16'
```

Comment:
I wanted to make the pipeline fail when custom rule added in #250 isn't added and called. The assumption here is the `grep 'custom-rule-name' test-format.txt` will find the line in `formatKotlin` task report and will make sure the offence was logged at the right spot:

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/36954793/189500463-566e3ceb-6fb9-4654-ad6c-3fb3af367e9c.png">